src/System.Diagnostics.StackTrace/src/System.Diagnostics.StackTrace.csproj

```diff
@@ -36,7 +36,7 @@
     <ProjectReference Include="..\..\System.Reflection.Metadata\src\System.Reflection.Metadata.csproj" />
     <ProjectReference Include="..\..\System.Collections.Immutable\src\System.Collections.Immutable.csproj" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsAOT)'!='true'">
+  <ItemGroup>
```

Comment:
You can delete `Condition="'$(TargetGroup)' == 'netcoreapp' or '$(TargetGroup)' == 'uap'"` above and fold the ItemGroups together.